common/src/IO/TextureLoader.cpp

```diff
@@ -58,6 +59,9 @@ namespace TrenchBroom {
             } else if (textureConfig.format.format == "idwal") {
                 TextureReader::PathSuffixNameStrategy nameStrategy(2, true);
                 return new IdWalTextureReader(nameStrategy, loadPalette(textureConfig));
+            } else if (textureConfig.format.format == "image") {
+                TextureReader::PathSuffixNameStrategy nameStrategy(3, true);
```

Comment:
@ericwa I think that's more a problem with the constructor itself. I do wonder however, is it correct to use the 3 last path components here? Doesn't that depend more on the game? Maybe I have to refactor this at some point.